pkg/components/state/pluggable.go

```diff
@@ -80,15 +81,15 @@ func (ss *grpcStateStore) Features() []state.Feature {
 }
 
 // Delete performs a delete operation.
-func (ss *grpcStateStore) Delete(req *state.DeleteRequest) error {
-	_, err := ss.Client.Delete(ss.Context, toDeleteRequest(req))
+func (ss *grpcStateStore) Delete(ctx context.Context, req *state.DeleteRequest) error {
+	_, err := ss.Client.Delete(ctx, toDeleteRequest(req))
```

Comment:
This doesn't use `ss.Context` anymore. Is that going to be a problem? CC: @mcandeia

Reply:
![image](https://user-images.githubusercontent.com/20457624/197667456-381b7441-eb0c-4043-b258-d2f34a92c98e.png)

I see that the member variable: `Context` in the GRPCConnector object is not assigned anywhere except after the initialization here, so I don't know what plans will be in the future.

Reply:
need to wait for @mcandeia  on that one

Reply:
OK.